stacks/nodejs-gatsby/devfile.yaml

```diff
@@ -0,0 +1,74 @@
+commands:
+  - exec:
+      commandLine: yarn add sharp
+      component: runtime
+      group:
+        isDefault: false
+        kind: build
+      workingDir: /project
+    id: add-sharp
+  - exec:
+      commandLine: yarn install --frozen-lockfile
+      component: runtime
+      group:
+        isDefault: false
+        kind: build
+      workingDir: /project
+    id: install
+  - exec:
+      commandLine: yarn build && yarn install --production --ignore-scripts --prefer-offline
+      component: runtime
+      group:
+        isDefault: false
+        kind: build
+      workingDir: /project
+    id: build
+  - composite:
+      commands:
+	- add-sharp
```

Comment:
needs tab here I think

Reply:
CI build should be catching this but isn't due to https://github.com/devfile/api/issues/533